packages/editor-ui/src/router.ts

```diff
@@ -130,15 +130,6 @@ export const routes = [
 			},
 			middleware: ['authenticated'],
 		},
-		beforeEnter: (to, _from, next) => {
-			const templatesStore = useTemplatesStore();
-			if (!templatesStore.hasCustomTemplatesHost) {
-				const id = Array.isArray(to.params.id) ? to.params.id[0] : to.params.id;
-				window.location.href = templatesStore.getWebsiteTemplatePageURL(id);
-			} else {
-				next();
-			}
-		},
 	},
 	{
 		path: '/templates/:id/setup',
```

Comment:
can you add a comment to this route? that we are using it for website. as last measure against it being deleted.

Reply:
Yup

Reply:
I mean the setup url since we will be redirecting to this